tests/functional/u/unbalanced_tuple_unpacking.py

```diff
@@ -106,3 +107,12 @@ def test_issue_559():
     from ctypes import c_int
     root_x, root_y, win_x, win_y = [c_int()] * 4
     return root_x, root_y, win_x, win_y
+
+
+class MyClass(NamedTuple):
+    first: float
+    second: float
```

Comment:
Could you add / modify one variable to include a default? Just to also test that.